lib/rubber/cloud/base.rb

```diff
@@ -10,7 +10,7 @@ def initialize(env, capistrano)
         @capistrano = capistrano
       end
 
-      def before_create_instance(instance_alias, role_names)
+      def before_create_instance(instance)
```

Comment:
I don't entirely understand this change. How can you have an instance before it's been created?